src/main/java/com/rocketden/main/dto/game/PlayerMapper.java

```diff
@@ -0,0 +1,48 @@
+package com.rocketden.main.dto.game;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.rocketden.main.dto.user.UserMapper;
+import com.rocketden.main.game_object.Player;
+import com.rocketden.main.model.User;
+import com.rocketden.main.util.Color;
+
+import org.modelmapper.convention.MatchingStrategies;
+
+public class PlayerMapper {
+
+    protected PlayerMapper() {}
+
+    public static PlayerDto toDto(Player player) {
+        if (player == null) {
+            return null;
+        }
```

Comment:
I saw you addressed this in your notes comments; maybe we can keep this method but continue to use the loose mapper object? That way if more fields are added they should be taken care of automatically, and if we need to do any complex stuff (like sorting) we could do it after using the mapper

Reply:
Yeah I think that makes sense -- I'll revert this back after reviewing the remaining PRs as well.

Reply:
Done! ✅

Reply:
Oh I mean I was totally fine with leaving the toDto method (I think it actually makes more sense too) but just keeping the mapper.map logic as the bulk of the implementation 

Either way though not too big of a deal so whatever you think makes sense

Reply:
Ah whoops lol -- maybe this is because I already switched it back, but since the code I provided doesn't do anything more that the `mapper.map` function does, I think it makes sense to keep just that until we actually need the `toDto` method.